frontend/src/components/material/MaterialCreateItem.vue

```diff
@@ -11,7 +11,6 @@
         ref="quantity"
         v-model="materialItem.quantity"
         dense
-        vee-rules="numeric"
```

Comment:
Is that no longer useful?

Reply:
Numeric only allows integer values. Our API allows float values. There's a built-in validation rule called "double" for float values. However, the "double" validation fails for empty values, which we explicitly allow.

Because we already set the html attribute type="number", the validation is not really needed, because the user cannot enter anything else than numbers anyway.